Cargo.toml

```
@@ -2,34 +2,31 @@
 name = "azure-init"
 version = "0.1.1"
 edition = "2021"
-
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+repository = "https://github.com/Azure/azure-init/"
+homepage = "https://github.com/Azure/azure-init/"
+license = "MIT"
+readme = "README.md"
+description = "A minimal guest agent implementation for provisioning Linux VMs on Azure."
```

```suggestion
description = "A reference implementation for provisioning Linux VMs on Azure."
```

I think we want to avoid the "agent" wording, especially "guest agent", which usually represents long running entities in the guest VM (such as WaLinuxAgent)

---

Makes sense. Then we would also need to update a wording in [README](https://github.com/Azure/azure-init/blob/02ade8608c75ae8e25f89b4d660ba73f0493cca3/README.md).

---

Done, thanks.